server/routes/review/visitsReviewListingController.ts

```diff
@@ -12,18 +14,49 @@ export default class VisitsReviewListingController {
   public view(): RequestHandler {
     return async (req, res) => {
       const { prisonName } = req.session.selectedEstablishment
-      const visitsReviewList = await this.visitNotificationsService.getVisitsReviewList(
+      // OK to cast these as express-validator sanitises to string[]
+      const bookedBy = Array.isArray(req.query.bookedBy) ? <string[]>req.query.bookedBy : []
```

Comment:
Bit of a limitation with `express-validator` is that although its `toArray()` sanitisation function at the bottom will have made these params a string array there doesn't seem to be a way to signal this to TypeScript so you still end up having to convince it we have a `string[]` 🤷‍♂️